wixcode-seo/wix-animations/TimeLine.service.json

```diff
@@ -0,0 +1,814 @@
+{ "name": "TimeLine",
+  "memberOf": "wix-animations",
+  "mixes": [],
+  "labels": [],
+  "location":
+    { "lineno": 1,
+      "filename": "timeline.js" },
+  "docs":
+    { "summary": "An animation timeline.",
+      "description":
+        [ "A timeline is used to compose animation sequences, built from [animations](#Animation),",
+          " which are played over time. You can synchronize multiple animations on one or more page",
+          " elements, control them as a whole, and precisely manage their timing." ],
+      "links": [],
+      "examples": [],
+      "extra":
+        {  } },
+  "properties": [],
+  "operations":
+    [ { "name": "add",
+        "labels": [],
+        "nameParams": [],
+        "params":
+          [ { "name": "target",
+              "type":
+                [ "$w.HiddenMixin",
```

Comment:
Why is the parameter a mixin? Does this mean all elements that import the mixin? Very confusing and requires explanation.